Neos.Utility.ObjectHandling/Classes/ObjectAccess.php

```diff
@@ -287,14 +283,18 @@ public static function getGettablePropertyNames($object)
         if (!isset(self::$gettablePropertyNamesCache[$className])) {
             foreach (get_class_methods($object) as $methodName) {
                 if (is_callable([$object, $methodName])) {
-                    if (substr($methodName, 0, 2) === 'is' && strlen($methodName) > 2) {
-                        $declaredPropertyNames[] = lcfirst(substr($methodName, 2));
-                    }
-                    if (substr($methodName, 0, 3) === 'get' && strlen($methodName) > 3) {
-                        $declaredPropertyNames[] = lcfirst(substr($methodName, 3));
-                    }
-                    if (substr($methodName, 0, 3) === 'has' && strlen($methodName) > 3) {
-                        $declaredPropertyNames[] = lcfirst(substr($methodName, 3));
+                    $methodNameLength = strlen($methodName);
```

Comment:
Not so sure about this restructuring. IMO this is now much harder to quickly grasp what this block is about